src/tools/spark.md

```diff
@@ -73,6 +64,8 @@ query API.
 
 ### Storage API
 
+> **⚠** Databricks is shut down as of January 2021. The storage API will not work on Google Dataproc.
```

Comment:
I wasn't sure about this. Is there an alternative for this API on dataproc or does it just happen to work?

Reply:
I don't know, maybe @akkomar or @acmiyaguchi does.

Reply:
It looks like Dataproc requires a special jar to be included: https://cloud.google.com/dataproc/docs/tutorials/bigquery-connector-spark-example

Reply:
cc @akkomar @BenWu

Reply:
```suggestion
```
Yeah, Storage API works irrespective of execution environment.